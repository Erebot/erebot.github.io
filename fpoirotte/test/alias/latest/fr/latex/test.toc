\select@language {french}
\contentsline {chapter}{\numberline {1}blah}{3}{chapter.1}
