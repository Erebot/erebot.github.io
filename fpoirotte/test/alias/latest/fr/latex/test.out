\BOOKMARK [0][-]{chapter.1}{blah}{}% 1
